lib/optimizer/configuration/break-up.js

```diff
@@ -193,15 +198,13 @@ function background(property, configuration, validator) {
           size.value = [previousValue, value];
           i -= 2;
         } else {
-          if (!positionSet)
-            position.value = [];
+          if (!positionSet) position.value = [];
```

Comment:
I'd rather wrap those in parentheses, e.g.

```
if (!positionSet) {
  position.value = [];
}
```

I found one-liners more difficult to read.

Reply:
As far as I can see there's no way to enforce the style I suggested, so nevermind, it's good as it is.